### PR TITLE
New Package: telegram-desktop and dependencies

### PR DESCRIPTION
**Pull request by Johnnynator**
Notes:
- Compilation for i686 and x86_64{,-musl} succeeded fine locally
- package tested successfully on x86_64-musl

### CODE REVIEW

**Comment by Johnnynator** [comment]
Compiled fine locally for all supported archs. Travis errors because it exceeded the max time limit of ~50min.

**Comment by skrzyp** [comment]
How's the status of that? From the last discussion on IRC I've thought it's generally okay-ish, but still no one accepted the PR.

**Comment by pullmoll** [comment]
We have a serious lag now because of many huge packages hitting the builders, changes like the libressl and its dependencies updates and failing builders were holding up things. I'm sure everyone of @voidlinux/pkg-committers does her best to catch up.

File: srcpkgs/gyp/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'gyp'
+pkgname=gyp
+version=20180218
```
Comment (Vaelatern) [inline srcpkgs/gyp/template]: Who decides this version?

Comment (Johnnynator) [reply srcpkgs/gyp/template]: Yep, I changed it. The project it self doesn't have any versioning. At least no visible one.

**Comment by pullmoll** [comment]
Provided #12055 is accepted, you could then make `gyp` download the tarball.
I really hate package templates fetching distfiles with `git clone` everytime I build them :)

**Comment by Johnnynator** [comment]
Updated gyp template to use the tarball

**Pull request closed without merging**